config/form_profile_mappings/21-22.yml

```diff
@@ -11,7 +11,6 @@ militaryInformation:
   serviceDateRange:
     from: [military_information, last_entry_date]
     to: [military_information, last_discharge_date]
-vaFileNumber: [veteran_information, va_file_number]
```

Comment:
`veteran_information` isn't a defined method for our class. The 21686c that defines the method for itself gets the information from EVSS. The full vile number is not a common mapping. Last 4 digits is more common. I've decided to just remove it since our form doesn't require it for submission.